SADL/AM-CDM-manufacturingProcess.sadl

```diff
@@ -0,0 +1,56 @@
+uri "http://kdl.ge.com/AM-CDM-manufacturingProcess" alias AM-CDM-manufacturingProcess version "$Revision:$ Last modified on   $Date:$".
+
+import "http://kdl.ge.com/AM-CDM-process".
+import "http://kdl.ge.com/AM-CDM-system".
+import "http://kdl.ge.com/AM-CDM-material".
+
+ManufacturingProcessStep (note "Planned process in which input material is transformed or modified") is a PlannedProcessStep,
```

Comment:
All classes should be written out as 
1. "...is a class.
OR
2. "...is a type of..."  
For ex:  ManufacturingProcessStep (note "Planned process in which input material is transformed or modified") is a type of PlannedProcessStep,